### PR TITLE
[better-enums] New port

### PR DESCRIPTION
**Pull request by Pravila00**
- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md)
- [ ] <del> The name of the port matches an existing name for this component on https://repology.org/ if possible, and/or is strongly associated with that component on search engines. </del>
- [ ] <del> Optional dependencies are resolved in exactly one way. For example, if the component is built with CMake, all `find_package` calls are REQUIRED, are satisfied by `vcpkg.json`'s declared dependencies, or disabled with [CMAKE_DISABLE_FIND_PACKAGE_Xxx](https://cmake.org/cmake/help/latest/variable/CMAKE_DISABLE_FIND_PACKAGE_PackageName.html) </del>
- [x] The versioning scheme in `vcpkg.json` matches what upstream says.
- [x] The license declaration in `vcpkg.json` matches what upstream says.
- [x] The installed as the "copyright" file matches what upstream says.
- [x] The source code of the component installed comes from an authoritative source.
- [ ] <del> The generated "usage text" is accurate. See [adding-usage](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/examples/adding-usage.md) for context. </del>
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is in the new port's versions file.
- [x] Only one version is added to each modified port's versions file.

### CODE REVIEW

File: ports/better-enums/portfile.cmake [hunk 1]
```diff
@@ -0,0 +1,11 @@
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO aantron/better-enums
+    REF 0.11.3
```
Comment (Cheney-W) [inline ports/better-enums/portfile.cmake:4]:
```suggestion
    REF ${VERSION}
```

File: ports/better-enums/portfile.cmake [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO aantron/better-enums
+    REF 0.11.3
+    SHA512 5997c74932803fb96beabbe029d80f6fdeab7c46f781a4e11ef775242d294dfd82ca05cac99787dd68a622db62510fd5533e9c0e85a62c7792c0dbe6237af6d9
+    HEAD_REF master
+)
+
+file(COPY "${SOURCE_PATH}/enum.h" DESTINATION "${CURRENT_PACKAGES_DIR}/include/better-enums")
+
+vcpkg_install_copyright(FILE_LIST "${SOURCE_PATH}/LICENSE.md")
```
Comment (Cheney-W) [inline ports/better-enums/portfile.cmake:11]:
```suggestion
vcpkg_install_copyright(FILE_LIST "${SOURCE_PATH}/LICENSE.md")

```

**Comment by Cheney-W** [comment]
Moving the PR to draft. Please set it to "ready for review" once changes have been applied.

**Comment by Pravila00** [comment]
@Cheney-W changes applied, thanks!

**Review by Cheney-W — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks!

**Pull request merged**